src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/appbar/AppBarIntegrationTests.cs

```diff
@@ -1961,7 +1961,7 @@ public async Task ValidateOverlayDCompTree()
 		[Description("Validates UIElement tree with an overlay-enabled app bar.")]
 		[TestProperty("TestPass:ExcludeOn", "WindowsCore")]
 		[TestProperty("HasAssociatedMasterFile", "True")]
-		public async Task ValidateOverlayUIETree()
+		public void ValidateOverlayUIETree()
```

Comment:
This test calls `ValidateUIElementTree` which is not implemented.

https://github.com/unoplatform/uno/blob/3903070e36668684ddceb31e895b610b022ed09d/src/Uno.UI.RuntimeTests/MUX/Helpers/ControlHelper.cs#L95-L104

Previously, it didn't fail because the method was `async` and not awaited here. This test should be skipped now.